scripts/install-handlebars-renderer.sh

```diff
@@ -2,7 +2,6 @@
 
 set -ev
 
-cd /opt
-git clone https://github.com/Hevienz/handlebars-renderer.git
-cd handlebars-renderer
-cargo install --path .
+# install handlerbars impl by python named pybars3
+# git@github.com:wbond/pybars3.git
+sudo pip install pybars3
```

Comment:
Need pip? then should add to doc.

Reply:
Yes. Need to render handlebars template files with pybar3.